rkt/completion.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdCompletion = &cobra.Command{
+		Use:   "completion SHELL",
+		Short: "Output shell completion code for the specified shell",
+		Long: `This command outputs completion code for the specified shell. The generated
+code must be evaluated to provide interactive completion of rkt sub-commands.
+This can be done by sourcing it from the .bash_profile.
+
+Save completion code in a home directory and then include it in .bash_profile
+script:
+
+	$ rkt completion bash > $HOME/.rkt.bash.inc
+	$ printf '
+# rkt shell completion
+source "$HOME/.rkt.bash.inc"
+' >> $HOME/.bash_profile
+	$ source $HOME/.bash_profile
+
+Alternatively, include the completion code directly into the launched shell:
+
+	$ source <(rkt completion bash)`,
+		ValidArgs: []string{"bash"},
+		Run:       runWrapper(newCompletion(os.Stdout)),
+	}
+
+	bashCompletionFunc = `__rkt_parse_image()
```

Comment:
I'm simply trying to follow Go naming convention for multiword identifiers, which dictates to use mixed caps instead of underscores: https://golang.org/doc/effective_go.html#mixed-caps.